luster_camera_width: 1920
luster_camera_height: 1080
luster_camera_fps: 30.0
luster_camera_gain: 10.0